File: Estructura_de_datos_Python/Busqueda_Lineal.py
numeros = [40,80,90,20,10]
# Pedir un numero para buscar
numero_Buscar = int(input("ingrese un numero a buscar: "))
encontrado = False

for i in range(len(numeros)):
    if numeros[i] == numero_Buscar:
        print("Número encontrado en la posición:", i)
        encontrado = True
        break
if not encontrado: 
    print("Número no encontrado en la lista.")

File: Estructura_de_datos_JavaScript/Busqueda_Lineal.js

const prompt = require("prompt-sync")(); 
let numeros = [10, 20, 30, 40, 50];
let numero_Abuscar = parseInt(prompt("Ingrese un número a buscar: "));

let encontrado = false;
// Busqueda de un número en el array
for (let i = 0; i < numeros.length; i++) {
    // compara cada elemento del array con el número a buscar
    if (numeros[i] === numero_Abuscar) {
        console.log("Número encontrado en la posición " + i);
        encontrado = true;
        break;
    }
}
// Si no se encontró el número, se muestra un mensaje
if (!encontrado) {
    console.log("El número no fue encontrado");
}
